programs/pkey/rsa_verify.c

```diff
@@ -56,11 +56,21 @@ int main( int argc, char *argv[] )
     FILE *f;
     int ret, c;
     size_t i;
+#if defined(MBEDTLS_PLATFORM_C)
+    mbedtls_platform_context platform_ctx;
+#endif
     mbedtls_rsa_context rsa;
     unsigned char hash[32];
     unsigned char buf[MBEDTLS_MPI_MAX_SIZE];
     char filename[512];
 
+#if defined(MBEDTLS_PLATFORM_C)
+    if( ( ret = mbedtls_platform_setup( &platform_ctx ) ) != 0 )
+    {
+        mbedtls_printf( " failed\n  ! mbedtls_platform_setup returned %d\n\n", -ret );
```

Comment:
The return value is displayed as a decimal here, but as hexadecimal at line 143. We should be self-consistent within the same program.